2024-05-05 16:07:58,167 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTP connection (1): 169.254.169.254:80
2024-05-05 16:08:19,209 FILE: creds.py IN: create_new_token MESSAGE: Ошибка получения iam_token: HTTPConnectionPool(host='169.254.169.254', port=80): Max retries exceeded with url: /computeMetadata/v1/instance/service-accounts/default/token (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x00000243D78784D0>, 'Connection to 169.254.169.254 timed out. (connect timeout=None)'))
2024-05-05 16:08:19,280 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTP connection (1): 169.254.169.254:80
2024-05-05 16:08:40,315 FILE: creds.py IN: create_new_token MESSAGE: Ошибка получения iam_token: HTTPConnectionPool(host='169.254.169.254', port=80): Max retries exceeded with url: /computeMetadata/v1/instance/service-accounts/default/token (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x00000243DAC523D0>, 'Connection to 169.254.169.254 timed out. (connect timeout=None)'))
2024-05-05 16:08:40,318 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTP connection (1): 169.254.169.254:80
2024-05-05 16:09:01,348 FILE: creds.py IN: create_new_token MESSAGE: Ошибка получения iam_token: HTTPConnectionPool(host='169.254.169.254', port=80): Max retries exceeded with url: /computeMetadata/v1/instance/service-accounts/default/token (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x00000243DAC61690>, 'Connection to 169.254.169.254 timed out. (connect timeout=None)'))
2024-05-05 16:09:01,349 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTPS connection (1): llm.api.cloud.yandex.net:443
2024-05-05 16:09:01,891 FILE: connectionpool.py IN: _make_request MESSAGE: https://llm.api.cloud.yandex.net:443 "POST /foundationModels/v1/tokenize HTTP/1.1" 200 None
2024-05-05 16:09:01,893 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTPS connection (1): api.telegram.org:443
2024-05-05 16:09:02,530 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getMe HTTP/1.1" 200 231
2024-05-05 16:09:02,532 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTPS connection (1): api.telegram.org:443
2024-05-05 16:09:03,374 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=1&timeout=20 HTTP/1.1" 200 412
2024-05-05 16:09:03,375 FILE: usersdb.py IN: create_database_users MESSAGE: DATABASE: База данных создана
2024-05-05 16:09:03,375 FILE: Bot.py IN: start MESSAGE: Обнаружение нового пользователя
2024-05-05 16:09:03,377 FILE: connectionpool.py IN: _new_conn MESSAGE: Starting new HTTPS connection (1): api.telegram.org:443
2024-05-05 16:09:04,062 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "POST /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/sendMessage?chat_id=1671928691&text=Choose+the+language&reply_markup=%7B%22keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22English%22%7D%5D%2C+%5B%7B%22text%22%3A+%22O%27zbek+tili%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%5Cu0420%5Cu0443%5Cu0441%5Cu0441%5Cu043a%5Cu0438%5Cu0439%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%5Cu049a%5Cu0430%5Cu0437%5Cu0430%5Cu049b%5Cu0448%5Cu0430%22%7D%5D%2C+%5B%7B%22text%22%3A+%22Deutsch%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%5Cu05e2%5Cu05d1%5Cu05e8%5Cu05d9%5Cu05ea%22%7D%5D%5D%2C+%22resize_keyboard%22%3A+true%7D HTTP/1.1" 200 294
2024-05-05 16:09:10,609 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=467934179&timeout=20 HTTP/1.1" 200 400
2024-05-05 16:09:10,609 FILE: Bot.py IN: send_logs MESSAGE: Получен файл debug
2024-05-05 16:09:10,779 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "POST /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/sendDocument?chat_id=1752643398 HTTP/1.1" 200 456
2024-05-05 16:09:30,690 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=467934180&timeout=20 HTTP/1.1" 200 23
2024-05-05 16:09:50,776 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=467934180&timeout=20 HTTP/1.1" 200 23
2024-05-05 16:10:10,868 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=467934180&timeout=20 HTTP/1.1" 200 23
2024-05-05 16:10:30,958 FILE: connectionpool.py IN: _make_request MESSAGE: https://api.telegram.org:443 "GET /bot6477745890:AAF_w2lcFKp4COkkJasYBtOFACU0ZH72HGE/getUpdates?offset=467934180&timeout=20 HTTP/1.1" 200 23
